.circleci/config.yml

```diff
@@ -21,25 +21,10 @@ jobs:
                 docker_layer_caching: true
             - checkout
             - run:
-                name: Pull existing docker image
+                name: Tag docker image
                 command: |
                     set +o pipefail
-                    docker pull houpy0829/chronon-ci:base || true
```

Comment:
Try change it to `docker pull houpy0829/chronon-ci:base--f87f50dc520f7a73894ae024eb78bd305d5b08e2` and see if it will work?